Readme.md

```diff
@@ -106,6 +106,15 @@ See `tools/docker/test-image/Dockerfile` for a full setup on Debian.
 - If you only want to embed v86 in a webpage you can use libv86.js. For
   usage, check out the [examples](examples/).
 
+For a full setup on Windows 10 WSL-2:
+
+- Install WSL-2 (Windows sub-linux system) from [here](https://docs.microsoft.com/en-us/windows/wsl/install)
+ - Install Windows sub-linux system ubuntu 20.04 from [here](https://www.microsoft.com/en-us/p/ubuntu-2004-lts/9n6svws3rx71?activetab=pivot:overviewtab)
+ - Install docsify-cli by using `npm i docsify-cli -g`
+ - Run `sudo dos2unix /usr/local/lib/node_modules/docsify-cli/bin/docsify`
```

Comment:
What is docsify used for?

Reply:
Hi sir:
        This is used to solve the problem of "/usr/bin/env: ‘node\r’: No such file or directory" when trying to build v86.

Reply:
That makes sense, but why is docsify needed in the first place?